CMakeLists.txt

```diff
@@ -15,11 +14,12 @@ project(otsdaq_mu2e_tracker VERSION 1.03.02)
 include(CetCMakeEnv)
 cet_cmake_env()
 
-cet_set_compiler_flags(DIAGS VIGILANT
- WERROR
- NO_UNDEFINED
- EXTRA_FLAGS -pedantic -Wno-unused-parameter -pthread
- )
+cet_set_compiler_flags(
+        DIAGS VIGILANT
+        WERROR  
+        NO_UNDEFINED
+        EXTRA_FLAGS -pedantic -Wno-unused-parameter -pthread -g -O0 -v 
```

Comment:
-g and -O are set by the prof or debug qualifier and should not be specified here.